Common.mk

```diff
@@ -132,7 +132,8 @@ else ifneq ($(IS_RELEASE_BRANCH_BUILD),)
 	RELEASE_TARGETS_OVERRIDE=release/release-branches/all
 
 	# avoid warnings when trying to read GIT_TAG file which wont exist when no release_branch is given
-	GIT_TAG=non-existent
+	# the GIT_TAG could be set during version-tracker's upgrade command, so no override
+	GIT_TAG?=non-existent
```

Comment:
i dont see why this is needed since youve added this target to the above allowlist, it shouldnt even fall into here?

Reply:
I tried without this "?" mark, but the command `make upgrade -C tools/version-tracker PROJECT=kubernetes-sigs/kind VERBOSITY=6` stuck at `/usr/bin/bash -c make -C /home/ubuntu/eks-anywhere-build-tooling/tools/version-tracker/eks-anywhere-build-tooling/projects/kubernetes-sigs/kind patch-repo` command, without output. I guess it's trying to wait for the non-exist tag

Reply:
I think the GIT_TAG=non-exist overrode the GIT_TAG set here: https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/kind/Makefile#L2